andela/banasko-rc
# #137156957 Getting Started Tour

### What does this PR do?

This PR shows a brief introductory walkthrough of how to use Reaction Commerce.

### Description of Task to be completed?

GIVEN: I just successfully signed up/register on the website
WHEN: I use the app for the first time I should automatically be introduced to a tour guide.
THEN: I should be able to get a brief tour of how to use the website, what I need to know about the features.
In addition, There should be a skip button incase the user doesn't want to complete the tour.

### How should this be manually tested?

Visit the app on heroku, register to see the tour or click the tour guide button

### Any background context you want to provide?

None

### What are the relevant pivotal tracker stories?

 As a new user, I want to see a brief tour of how to use the web app and the important feature so that I have sufficient knowledge on how to continue to use the website.

### Screenshots (if appropriate)

### Questions:

None
